concourse_ci/manifest.json

```diff
@@ -0,0 +1,23 @@
+{
+    "categories": [
+        "Collaboration"
+    ],
+    "guid": "054cc9fb-01c4-4f05-98b5-fae828746787",
+    "is_public": true,
+    "maintainer": "help@datadoghq.com",
+    "public_title": "Datadog-Concourse-CI Integration",
+    "support": "core",
+    "supported_os": [
+        "linux",
+        "mac_os",
+        "windows"
+      ],
+    "type": "check",
+    "manifest_version": "1.0.2",
+    "name": "concourse_ci",
+    "display_name": "Concourse CI",
+    "short_description": "Collect metrics emitted from Concourse CI.",
+    "metric_prefix": "concourse.ci.",
+    "metric_to_check": "concourse.ci.goroutines",
```

Comment:
Is this metric always guaranteed? If I'm not mistaken the tile will show as broken if this metric is not being collected.

Reply:
From my local tests, I noticed that this metric is emitted when the emmiter is configured and it periodically sends data even if no pipelines or jobs are created.

https://github.com/concourse/atc/blob/master/metric/periodic.go#L126-L131

Reply:
This is a default go metric, I'd be surprised if it was ever not emitted.